api/db/migrations/20210721084659_add-birthCountry-in-certification-course-table.js

```diff
@@ -0,0 +1,12 @@
+
+exports.up = async function(knex) {
+  await knex.schema.table('certification-courses', (table) => {
+    table.string('birthCountry');
```

Comment:
c'est pas tout de l'ajouter dans la migration, il faut donc aussi penser à recopier la donnée vers le certification course lorsqu'on lance un test de certification pour être ISO partout.
A faire ici -> api/lib/domain/models/CertificationCourse.js:55

Reply:
Et puis je pense qu'il y aurait quelque chose à faire ici aussi pour la prescription de certification SCO -> api/lib/domain/usecases/enroll-students-to-session.js:30

Reply:
Attends attends, ça vient dans un prochain commit :sweat_smile:

Reply:
@laura-bergoens En fait pour les schooling registrations, on ne gère que les candidats Français ? 

Dans le `enroll-candidate` on fait 
```const scoCertificationCandidates = students.map((student) => {
    return new SCOCertificationCandidate({
      firstName: student.firstName.trim(),
      lastName: student.lastName.trim(),
      birthdate: student.birthdate,
      birthINSEECode: student.birthCityCode,
      sex: student.sex,
      sessionId,
      schoolingRegistrationId: student.id,
    });
  });```
  
  Du coup le birthInseeCode de ce type de candidate correspond toujours à une birthCity, donc le candidat est toujours née en France, correct ?

Reply:
La méthode static du CertificationCourse

```
static from({ certificationCandidate, challenges, verificationCode, maxReachableLevelOnCertificationDate }) {
    return new CertificationCourse({
      userId: certificationCandidate.userId,
      sessionId: certificationCandidate.sessionId,
      firstName: certificationCandidate.firstName,
      lastName: certificationCandidate.lastName,
      birthdate: certificationCandidate.birthdate,
      birthPostalCode: certificationCandidate.birthPostalCode,
      birthINSEECode: certificationCandidate.birthINSEECode,
      birthCountry: certificationCandidate.birthCountry,
      sex: certificationCandidate.sex,
      birthplace: certificationCandidate.birthCity,
      externalId: certificationCandidate.externalId,
      isV2Certification: true,
      challenges,
      verificationCode,
      maxReachableLevelOnCertificationDate,
    });
  }
 ```
 
 prend un certif candidat en paramètre. Celui ci voit ses infos CPF misent à jour lors de l'import via l'ods
 
 ```
   if (version === '1.5') {
    const cpfBirthInformation = await certificationCpfService.getBirthInformation({
      ...certificationCandidate,
      certificationCpfCityRepository,
      certificationCpfCountryRepository,
    });

    if (cpfBirthInformation.hasFailed()) {
      throw new CpfBirthInformationValidationError(cpfBirthInformation.message);
    }

    certificationCandidate.updateBirthInformation(cpfBirthInformation);
  }
 ```
 
 Mais les candidats sco ne passent jamais par le certificationCpfService, du coup ce type de candidat n'a qu'un INSEECode, jamais de birthCity ou de birthCountry

Reply:
Question: On ajoute le birthCountry forcé à 'FRANCE' dans `SCOCertificationCandidate` ?